src/Glpi/Kernel/Kernel.php

```diff
@@ -150,7 +150,10 @@ protected function configureContainer(ContainerConfigurator $container): void
 
     protected function configureRoutes(RoutingConfigurator $routes): void
     {
-        //  Global core controllers
+        // Item types controllers, usually they override generic or legacy ones
+        $routes->import($this->getProjectDir() . '/src/Glpi/Controller/ItemType', 'attribute');
```

Comment:
Why do we need this?

Reply:
We don't, the issue was fixed by checking the cache :p